src/components/screens/InfoCard.js

```diff
@@ -32,8 +32,7 @@ const contactView = (contact) => (
       <InfoBox>
         <Icon xml={phoneIcon(colors.primary)} style={styles.margin} />
         <TouchableOpacity
-          accessibilityLabel="Telefonnummer"
-          accessibilityHint="Navigieren zu anruf option"
+          accessibilityLabel={`(Telefonnummer) (${contact.phone}) (Taste) (Wechselt zur Telefon-App)`}
```

Comment:
nice breaks. around `${contact.phone}` could be too much, as the text before and the text after is already in `(...)`?